tests/integration/test_model.py

```diff
@@ -835,11 +835,40 @@ async def test_wait_for_idle_with_exact_units_scale_down(event_loop):
         await app.destroy_units(*two_units_to_remove)
 
         # assert that the following wait is not returning instantaneously
-        starttime = time.time()
+        start_time = time.time()
         await model.wait_for_idle(timeout=5 * 60, wait_for_exact_units=1)
-        endtime = time.time()
+        end_time = time.time()
```

Comment:
Bit of a knitpick, but it's generally recommended to use time.perf_counter() when timing duration. time.time() gives the current unix time and is more intended to get the current time, rather than measuring short durations